mkosi

```diff
@@ -3396,7 +3396,7 @@ def run_qemu(args):
         die("Couldn't find QEMU/KVM binary")
 
     # Look for UEFI firmware blob
-    for firmware in ('/usr/share/edk2/ovmf/OVMF_CODE.fd',):
+    for firmware in ('/usr/share/edk2/ovmf/OVMF_CODE.fd','/usr/share/ovmf/ovmf_code_x64.bin'):
```

Comment:
hmm, that file appears to be appropriate only for x86-64, no? i figure this should be conditionalized appropriately...

Reply:
Indeed.
The `ovmf` package comes with two firmware blobs.
I’m now selecting the right one depending on the architecture.